pkg/cli/registry/login/login.go

```diff
@@ -293,10 +298,11 @@ func (o *LoginOptions) Run() error {
 	}
 
 	ctx := &containertypes.SystemContext{AuthFilePath: o.ConfigFile}
-	if err := dockerconfig.SetAuthentication(ctx, o.HostPort, o.Credentials.Username, o.Credentials.Password); err != nil {
+	credentialLocation, err := dockerconfig.SetCredentials(ctx, o.HostPort, o.Credentials.Username, o.Credentials.Password)
+	if err != nil {
 		return err
 	}
 
-	fmt.Fprintf(o.Out, "Saved credentials for %s\n", o.HostPort)
+	fmt.Fprintf(o.Out, "Saved credentials for %s into %s\n", o.HostPort, credentialLocation)
```

Comment:
I have changed the standart output to include the location where the credentials were saved, so there should not be any doubts where the creds ended up

Unfortunately it is not easy to obtain the location for the commands that just lookup the credentials (and potentially log it), unless we replicate the lookup logic. Nevertheless since we are falling back to ~/.docker/config.json credentials when podman credentials are not found, it should not be a big issue. The only small issue is a conflict, but a user that is purposefully using both docker and podman should be aware that such state can occur and debug the auth files, when login fails.